### ISSUE TITLE
Problem with ZZZ check-in

### ISSUE THREAD

**Issue by Annie3Starlight** [Triginarsa/hoyolab-daily-checkin-bot#1]
When checking in, I receive two errors, which say:
"An error occurred: Error fetching sign info: Error: Failed to get sign info: 
{"data":null,"message":"网络出小差了，请稍后重试~","retcode":-500001}"
and
"An error occurred: Sign info not found for /insert name/"
Been happening for several days now. Have absolutely no idea what's up with that, but for now I'll have to check in manually

**Comment by Triginarsa** [comment]
Hi @Annie3Starlight , I've pushed the new code to fix this issue. Please use the updated code.

Note: You can still use the old cookie data.

**Comment by Annie3Starlight** [comment]
Thank you, that was fast
So far, this is the most serious bug that has occurred while I was using the bot. Previous bugs were mostly "something is wrong with your cookies" basically. It could happen to any game and usually would be fine the next day, so eh, works for me
I went to change the script as soon as I got home. I'll check the results when it's time for the next check-in. If it's all good, I'll close the issue

Edit: sorry, accidentally closed because I wanted to test something
Edit 2: now closing for real
